### ISSUE TITLE
triple backticks are not escaped in error messages

### ISSUE THREAD

**Issue by bmintz** [Gorialis/jishaku#8]
message: <code>ec/jsk py '\`\`\`foo\`\`\`</code>
## Expected response
```
``​`py
Traceback (most recent call last):
  File "<repl-x session>", line 9
    '```foo
          ^
SyntaxError: EOL while scanning string literal
``​`​
```
Image: soon™
​
## Actual response:
I can only provide a screenshot since the difference is invisible from looking at the source of the message
![screenshot of the buggy behavior](https://user-images.githubusercontent.com/538336/42744328-474b9588-8891-11e8-8ae6-a3a7f1606256.png)

This can be fixed by escaping the \`\`\`. [A routine that escapes these](https://github.com/khazhyk/dango.py/blob/69f4af8e8382de99f1683ac12b4f70119da9c30b/plugins/common/utils.py#L106-L125) is provided as part of spoo.py.

**Comment by Gorialis** [comment]
I will fix this with other sanitization in jsk 1.0.0

**Comment by Gorialis** [comment]
This is now [fixed](https://github.com/Gorialis/jishaku/blob/350b5176703790dded51280acd97e767a7fa4591/jishaku/exception_handling.py#L37) as of jishaku 1.0.1
